kazakh/wca-regulations.md

```diff
@@ -0,0 +1,518 @@
+# <wca-title> Wca Ережелерi
+
+<version>нұсқасы: 1 қаңтар 2020 ж.
+
+
+## Ескертулер
+
+### Аударма
+
+Бұл ресми WCA ережелері емес, бірақ тегін аударма. Күдік туындаған жағдайда бастапқы Ережеге жүгініңіз. Аудармада қателер болуы мүмкін! Аударуға қатысты сұрақтар мен ескертулерді [iarystanbekov@worldcubeassociation.org](mailto:iarystanbekov\@worldcubeassociation.org)электрондық поштасына жіберіңіз.
+
+Кейбір терминдер әдейі аударылмаған, өйткені қазақ тілінде сәйкес сөздер жоқ, немесе тіпті орыс жарыстарында ағылшын атаулары қолданылады.Оларға мыналар жатады: скрамблер (жұмбақтарды араластыратын адам), square-1, Clock (жұмбақтардың аттары) және т.б.
+
+Аударманы: [Ильяс Арыстанбеков](https://www.worldcubeassociation.org/persons/2014ARYS01), Сұлтанбибарыс Омар (https://www.worldcubeassociation.org/persons/2017SULT02) жүргізді.

```

Comment:
I don't think the credit section should be included on the Regulations file.

Reply:
> I don't think the credit section should be included on the Regulations file.

Given that it doesn't fit on the translations page anymore, I think on in the document itself is best. We could standardize the section, but I think allowing it directly inline for now is sufficiantly good.

Reply:
That makes sense.